src/components/newEvent/Overview.tsx

```diff
@@ -0,0 +1,247 @@
+import {useForm} from "react-hook-form"
+import { z, ZodType } from "zod"; // Add new import
+import { FieldError, UseFormRegister } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useContext, useState } from "react";
+import { newEventContext } from "@/app/events/new/page";
+
+interface FormData {
+    eventName: string;
+    startsAt: string;
+    startsAtTime: string;
+    endsAt: string;
+    endsAtTime: string;
+    type: string;
+    imageUrl: string;
+  };
+
+interface FormFieldProps {
+    className: string;
+    label:string;
+    type: string;
+    placeholder: string;
+    name: ValidFieldNames;
+    register: UseFormRegister<FormData>;
+    error: FieldError | undefined;
+    valueAsNumber?: boolean;
+  };
+
+
+type ValidFieldNames =
+  | "eventName"
+  | "startsAt"
+  | "startsAtTime"
+  | "endsAt"
+  | "endsAtTime"
+  | "type"
+  | "imageUrl";
+
+
+const FormField: React.FC<FormFieldProps> = ({
+  className,
+  label,
+  type,
+  placeholder,
+  name,
+  register,
+  error,
+  valueAsNumber,
+}) => {
+    const strongClassName = (label=="empty") ? "w-fit text-transparent" : "w-fit"
```

Comment:
I guess this project does not have any format/lint tool. I would install it with https://biomejs.dev/guides/getting-started/

Reply:
I installed and run the lint. Idk if it was supposed to rearrange the ?: syntax in new lines, but it didn't. Only complained that I used "==" instead of "==="

Reply:
probably the format is not working. let's fix this together?

Reply:
Ok, for some reason it started working after I removed biome.json